src/index.html

```diff
@@ -1,13 +1,334 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BOSE</title>
+    <link
+      rel="shortcut icon"
+      href="./images/favicon.png"
+      type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__header">
+      <div class="container">
+        <div class="header__content">
+          <div class="nav">
+            <a href="#" class="nav__logo">
+              <img
+                src="./images/Boss_img/logo.svg"
+                alt="BOSE"
+                class="nav__logo-img"
+              >
+            </a>
+
+            <div class="nav__icons">
+              <a href="tel:+1 234 555-55-55" class="nav__call icon--call icon"></a>
+
+              <a href="#menu" class="nav__menu icon--menu icon"></a>
+            </div>
+          </div>
+
+          <h1 class="header__title">
+            The world shades.
+            <br>Your music shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="menu page__menu" id="menu">
```

Comment:
It's weird that I click on menu at the right side of the screen and it comes out from the left side

Reply:
Also, checklist 18th item
![image](https://user-images.githubusercontent.com/37161315/170693686-17441c44-789c-4f84-8565-77dc7f7575f9.png)